### PR TITLE
Frontend UI - stock discovery - create horizontal scrolling menus

### PR DESCRIPTION
**Pull request by DaraghK93**
created the stock container component for stock discovery. This will break the stock discovery page up into various rows and display them as horizontally scrolling menus, like the netflix homescreen.

Screenshot 2022-10-20 at 21 06 36

I had to slightly change the styling of the ticker cards as well, done inline on the TickerCard.js file.

### CODE REVIEW

File: amplify/backend/function/expressServer/src/config/default.json
Comment (BearachB) [inline amplify/backend/function/expressServer/src/config/default.json:4]: Not quite sure why this is showing up as a change, as this new URI is already merged to master. Weird but at least it's not changing

**Review by BearachB — APPROVED** [review]
Looks good overall Daragh! I've left some minor comments, nothing big enough to hold off the PR though.

One more minor comment - Might want to add a bit of spacing between the search bar and the title of "Companies That Are Good For The Environment" (see screenshot). Also, we need to come up with some catchier titles for those categories, not sure what they'd be but need to have a think about it.

image

**Review by WarrenK-design — APPROVED** [review]
Hi Daragh,

Looks like its coming together nicely, still needs a bit of work but accepting so we can show it tomorrow. Adding top movers, top losers, etc. and fixing the cards a bit more but think Becks nearly done with some of that. 

I did commit to it to just get rid of that annoying key warning so do a pull if you change anything before merging. 

image
image

**Pull request merged**
